data/siggraph-2019/10.1145-3306346.3323009.json

```diff
@@ -0,0 +1,206 @@
+        "##": "5 = no dependencies or some dependencies but trivial build/integration with the project",
+        "Dependencies score {0=NA, 1,2,3,4,5}": 5,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the build/configuration stage",
+        "##": "0 = no instructions or unable to build",
+        "##": "1 = many issues during the build or configure",
+        "##": "5 = no problem nor issue during the build/configure",
+        "Build/configure score {0=NA, 1,2,3,4,5}": 4,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the bug fixes (if any)",
+        "##": "0 = no bugs or irrelevant",
+        "##": "1 = many bugs, unable to fix them",
+        "##": "5 = just very minor bugs but very easy to fix",
+        "Fixing bugs score (if any) {0=NA, 1,2,3,4,5}": 0,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the replicability",
+        "##": "0 = N/A",
+        "##": "1 = I was not able to generate something or unable to analyze the output",
+        "##": "5 = I was able to reproduce most paper results and the remaining ones could be trivially obtained",
+        "Replicate paper results score {0=NA, 1,2,3,4,5}": 5,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the adaptability",
+        "##": "0 = N/A",
+        "##": "1 = The code is very specific to the paper contribution and cannot be used in other projects",
+        "##": "5 = The code can be easily used for other projects (as it is) or can be easily extended to other contexts",
+        "Adaptability score to other contexts {0=NA, 1,2,3,4,5}": 5,
+
+        "##": "Time spent for the replicability test (from the download to the first successful run)",
+        "##": "0 = less than 10 min",
+        "##": "1 = between 10 and 20 min",
+        "##": "...",
+        "##": "10 = more than 100 min",
+        "Time spent for the test (code download to first successful run, [0,10], 10min slots, 100min max)": 0,
+
+        "##": "Detailed instructions (especially if you had to fix some bugs before being able to run a test)",
+        "Build instructions/comments": "There is no documentation to build and run the code, but it is quite easy anyway since it relies on PBRT. I pulled PBRT v3 (aaa552a4b9cbf9dccb71450f47b268e0ed6370e2) with recursive dependencies, replaced PBRT files with those provided in supplementary materials, ran CMake (works directly), and added integrators/optmis.cpp and core/guiding.cpp in imgtool, bsdftest, pbrt_test, and pbrt_exe Visual Studio projects (you may as well edit the CMakeLists.txt). Compiling worked directly. Example scenes with pbrt scripts already set up to run the proposed MIS strategy are given in supplementary materials, for instance in supplemental/results/dining-room/scene (run pbrt.exe dining-room.pbrt as usual). It produces results comparables to those advertised. \n\nAdditional scripts to reproduce plots (some rely on Mathematica, others on Matlab ; I did not evaluate them) and to compare image results from different methods are provided.",
```

Comment:
```suggestion
        "Build instructions/comments": "There is no documentation to build and run the code, but it is quite easy anyway since it relies on PBRT. I pulled PBRT v3 (aaa552a4b9cbf9dccb71450f47b268e0ed6370e2) with recursive dependencies, replaced PBRT files with those provided in supplementary materials, ran CMake (worked directly), and added integrators/optmis.cpp and core/guiding.cpp in imgtool, bsdftest, pbrt_test, and pbrt_exe Visual Studio projects (you may as well edit the CMakeLists.txt). Compiling worked directly. Example scenes with pbrt scripts already set up to run the proposed MIS strategy are given in supplementary materials, for instance in supplemental/results/dining-room/scene (run pbrt.exe dining-room.pbrt as usual). It produces results comparable to those advertised. \n\nAdditional scripts to reproduce plots (some rely on Mathematica, others on Matlab ; I did not evaluate them) and to compare image results from different methods are provided.",
```

Reply:
J'approuve! :p